test/05-device-state.spec.ts

```diff
@@ -230,24 +234,30 @@ describe('deviceState', () => {
 			Promise.resolve('172.17.0.1'),
 		);
 
-		stub(deviceState.applications.images, 'inspectByName').callsFake(() => {
-			const err: any = new Error();
+		// @ts-expect-error Assigning to a RO property
```

Comment:
We recently agreed to start using ts-expect-error however, @Page- had mentioned some valid points about the use of ts-expect-error vs ts-ignore. I am convinced for situations such as this where we are just suppressing an error to achieve the right conditions for a test then ts-ignore is more suitable. This directive is found in multiple places in the PR so if you make the change I'd apply to all the other places as well.